8-typography.rst

```diff
@@ -1542,6 +1542,48 @@ Citations
 				<cite>—<i epub:type="se:name.publication.book">The Rubaiyat of Omar Khayyam</i></cite>
 			</blockquote>
 
+Verses and Chapters of the Bible
+================================
+
+#.	Citations of passages from the Bible should include the name of the book, followed by the chapter number and the verse number.
+
+	#.	The chapter and the verse numbers should be separated by a colon.
+
+	#.	All chapter and verse numbers should be written in Arabic numerals. Similarly, if a book being cited is a “numbered” book, the number should also be written in Arabic numerals.
+
+	.. class:: wrong
+
+		.. code:: html
+
+			<blockquote>
+				<p>“Though I speak with the tongues of men and of angels, and have not charity, I am become as sounding brass, or a tinkling cymbal.”</p>
+				<cite>—<span epub:type="z3998:roman">I</span> Corinthians <span epub:type="z3998:roman">XIII</span> 1</cite>
+			</blockquote>
+
+	.. class:: corrected
+
+		.. code:: html
+
+			<blockquote>
+				<p>“Though I speak with the tongues of men and of angels, and have not charity, I am become as sounding brass, or a tinkling cymbal.”</p>
+				<cite>—1 Corinthians 13:1</cite>
+			</blockquote>
+
+#.	If an entire chapter, instead of a particular verse, is being cited, then the citation should include the name of the book followed by the chapter number.
+
```

Comment:
We should add something saying to use en dashes for verse ranges (this is already addressed elsewhere as you noted, but we can repeat here for clarity), and addressing your comment on the list: `Emma, I typically see a comma separating the disconnected verses in the same chapter--so in your case Acts 8:9, 18. And if there were citations to some verses in the same chapter and some in different chapters, it might look like Acts 8:9, 18; 9:1, 3. I can put that into the PR as well if we want to settle on that standard.`